modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -295,6 +295,27 @@
 		</constant>
 	</constants>
 	<annotations>
+		<annotation name="@abstract">
+			<return type="void" />
+			<description>
+				Marks a class or a method as abstract.
+				An abstract class is a class that cannot be instantiated directly. Instead, it is meant to be inherited by other classes. Attempting to instantiate an abstract class will result in an error.
+				An abstract method has no implementation. Therefore, a newline is expected after the function header. This defines a contract that inheriting classes must conform to, because the method signature must be compatible when overriding.
```

Comment:
```suggestion
				An abstract method is a method that has no implementation. Therefore, a newline or a semicolon is expected after the function header. This defines a contract that inheriting classes must conform to, because the method signature must be compatible when overriding.
```